src/openrct2/scripting/ScMap.hpp

```diff
@@ -168,6 +191,7 @@ namespace OpenRCT2::Scripting
             dukglue_register_method(ctx, &ScMap::getTile, "getTile");
             dukglue_register_method(ctx, &ScMap::getEntity, "getEntity");
             dukglue_register_method(ctx, &ScMap::getAllEntities, "getAllEntities");
+            dukglue_register_method(ctx, &ScMap::getStaffPatrolArea, "getStaffPatrolArea");
```

Comment:
Why would you make this part of map and not part of entity?

Reply:
Adding getPatrolArea as a dukglue method on the SCStaff type does make more sense; making that modification now. I'm also not clear on which part of the commit message guide I'm not following; just the keep it in present tense part?